src/query/procedure/module.cpp

```diff
@@ -1326,10 +1327,25 @@ std::optional<std::pair<ModulePtr, const T *>> MakePairIfPropFound(const ModuleR
     }
   };
   auto result = FindModuleNameAndProp(module_registry, fully_qualified_name, memory);
-  if (!result) return std::nullopt;
+  if (!result) {
+    return std::nullopt;
+  }
   auto [module_name, prop_name] = *result;
   auto module = module_registry.GetModuleNamed(module_name);
-  if (!module) return std::nullopt;
+  if (!module) {
+    // Check for possible callable aliases.
+    const auto maybe_valid_alias = gCallableAliasMapper.FindAlias(std::string(fully_qualified_name));
+    if (maybe_valid_alias) {
+      result = FindModuleNameAndProp(module_registry, *maybe_valid_alias, memory);
```

Comment:
Perhaps put this in a helper function and call it twice, instead of having duplicate logic.

Reply:
I am really not sure what are you referring to here, where is the logic duplicated?`
FindModuleNameAndProp` and `FindAlias` are not always used together, the FindAlias is only called the if the unmapped, original procedure name is not found in the module registry.

Reply:
I am also not sure what you meant andreja but I would also separate this into smaller functions, having 3 nested if statements can always be improved